plexpy/datatables.py

```diff
@@ -70,6 +70,39 @@ def ssp_query(self,
         else:
             grouping = False
 
+        # Build join parameters
+        if join_types:
+            counter = 0
+            for join_type in join_types:
+                if join_type.upper() == 'LEFT OUTER JOIN':
+                    join_item = 'LEFT OUTER JOIN %s ON %s = %s ' % \
+                                (join_tables[counter], join_evals[counter][0], join_evals[counter][1])
+                elif join_type.upper() == 'JOIN' or join_type.upper() == 'INNER JOIN':
+                    join_item = 'JOIN %s ON %s = %s ' % \
+                                (join_tables[counter], join_evals[counter][0], join_evals[counter][1])
+                else:
+                    join_item = ''
+
+                counter += 1
+                join += join_item
+
+        # Build custom where parameters
+        if custom_where:
+            for w in custom_where:
+                c_where += w[0] + ' = ? AND '
+
+                # The order of our args changes if we are grouping
+                #if grouping:
+                #    args.insert(0, w[1])
+                #else:
+                #    args.append(w[1])
+
+                # My testing shows that order of args doesn't change
+                args.append(w[1])
```

Comment:
Need to confirm this.

I think it should be okay since custom where is created before the search filter args now.